examples/ReReleaseNuGet/Readme.md

```diff
@@ -0,0 +1,75 @@
+# Re-releasing Profiler.ASPNetCore package
```

Comment:
Maybe a different title: Properly reference Microsoft.ApplicationInsights.Profiler.AspNetCore in another NuGet package